content/INTRODUCTION.md

```diff
@@ -0,0 +1,25 @@
+**Requirements**: Knowledge in PostgreSQL, PHP, and Object-Oriented Programming in PHP 
+
+**Duration**:
+
+**Standards**: K-12 Computer Science Framework Core Concepts and Practices
+* Algorithms and Programming
+    - Collaborating Around Computing
+    - Recognizing and Defining Computational Problems
+    - Developing and Using Abstractions
+    - Creating Computational Artifacts
+    - Testing and Refining Computational Artifacts
+
+## Database Management in PHP
+
+Database management in PHP lets you build PHP programs that define, construct, and manipulate a database with PostgreSQL. This course covers the basics of performing PostgreSQL programming in PHP. It requires `PG PHP SQL extension` for PHP and PostgreSQL to communicate.
```

Comment:
Please remove this statement __`It requires PG PHP SQL extension for PHP and PostgreSQL to communicate.`__ since we will be using `PDO` in this course.

Reply:
I think we can have a short definition of PDO in here... Please use this as reference. Thanks.

> The PHP Data Objects (PDO) extension defines a lightweight, consistent interface for accessing databases in PHP. Each database driver that implements the PDO interface can expose database-specific features as regular extension functions.